platform-sdk/swirlds-platform-core/src/testFixtures/java/com/swirlds/platform/test/fixtures/crypto/PreGeneratedX509Certs.java

```diff
@@ -63,74 +37,6 @@ private PreGeneratedX509Certs() {
         throw new IllegalStateException("Utility class");
     }
 
-    /**
-     * Generates a set of X.509 certificates for testing purposes.
-     * <p>
-     * The method must be called from within swirlds-common for the path to resolve to the right location.
-     *
-     * @param numCerts the number of certificates to generate
-     * @param random   the random number generator to use
-     */
-    public static void generateCerts(final int numCerts, @NonNull final Random random)
```

Comment:
This method is not used (anymore) and the `PATH_TO_RESOURCES` constant was not correct (anymore?). Hence I removed this completely.